site/_posts/2017-01-16-destroy-the-monolith.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Last Week in ManageIQ: Destroy the Monolith"
+author: durandom
+date: 2017-01-16 09:24:51 +01:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Moin](https://en.wikipedia.org/wiki/Moin), [durandom][https://github.com/durandom/] from the providers team here, 
```

Comment:
It looks like the hard brackets for the `durandom` url need to be swapped for `()`